content/Rust-1.91.0.md

```diff
@@ -0,0 +1,170 @@
+track raw pointers. With this release, we are adding a warn-by-default lint on raw
+pointers to local variables being returned from functions. For example, code like this:
+
+```rust
+fn f() -> *const u8 {
+    let x = 0;
+    &x
+}
+```
+
+will now produce a lint:
+
+```
+warning: a dangling pointer will be produced because the local variable `x` will be dropped
+ --> src/lib.rs:3:5
+  |
+1 | fn f() -> *const u8 {
+  |           --------- return type of the function is `*const u8`
+2 |     let x = 0;
+  |         - `x` is part the function and will be dropped at the end of the function
+3 |     &x
+  |     ^^
+  |
+  = note: pointers do not have a lifetime; after returning, the `u8` will be deallocated
+    at the end of the function because nothing is referencing it as far as the type system is
+    concerned
+  = note: `#[warn(dangling_pointers_from_locals)]` on by default
+```
+
+Note that the code above is not unsafe, as it itself doesn't perform any dangerous
+operations. Only dereferencing the raw pointer after the function returns would be
+unsafe. We expect future releases of Rust to add more functionality helping authors
+to safely interact with raw pointers, and with unsafe code more generally.
+
+### Stabilized APIs
+
+- [`Path::file_prefix`](https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.file_prefix)
+- [`AtomicPtr::fetch_ptr_add`](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicPtr.html#method.fetch_ptr_add)
+- [`AtomicPtr::fetch_ptr_sub`](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicPtr.html#method.fetch_ptr_sub)
+- [`AtomicPtr::fetch_byte_add`](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicPtr.html#method.fetch_byte_add)
+- [`AtomicPtr::fetch_byte_sub`](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicPtr.html#method.fetch_byte_sub)
+- [`AtomicPtr::fetch_or`](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicPtr.html#method.fetch_or)
+- [`AtomicPtr::fetch_and`](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicPtr.html#method.fetch_and)
+- [`AtomicPtr::fetch_xor`](https://doc.rust-lang.org/stable/std/sync/atomic/struct.AtomicPtr.html#method.fetch_xor)
+- [`{integer}::strict_add`](https://doc.rust-lang.org/stable/std/primitive.u32.html#method.strict_add)
+- [`{integer}::strict_sub`](https://doc.rust-lang.org/stable/std/primitive.u32.html#method.strict_sub)
+- [`{integer}::strict_mul`](https://doc.rust-lang.org/stable/std/primitive.u32.html#method.strict_mul)
+- [`{integer}::strict_div`](https://doc.rust-lang.org/stable/std/primitive.u32.html#method.strict_div)
+- [`{integer}::strict_div_euclid`](https://doc.rust-lang.org/stable/std/primitive.u32.html#method.strict_div_euclid)
+- [`{integer}::strict_rem`](https://doc.rust-lang.org/stable/std/primitive.u32.html#method.strict_rem)
+- [`{integer}::strict_rem_euclid`](https://doc.rust-lang.org/stable/std/primitive.u32.html#method.strict_rem_euclid)
```

Comment:
The div/rem entries all link to `u32` documentation that says overflow cannot happen and these methods exist for consistency. Perhaps it would be better to link to `i32` documentation which does have an overflow case, for less chance of looking silly?

```suggestion
- [`{integer}::strict_div`](https://doc.rust-lang.org/stable/std/primitive.i32.html#method.strict_div)
- [`{integer}::strict_div_euclid`](https://doc.rust-lang.org/stable/std/primitive.i32.html#method.strict_div_euclid)
- [`{integer}::strict_rem`](https://doc.rust-lang.org/stable/std/primitive.i32.html#method.strict_rem)
- [`{integer}::strict_rem_euclid`](https://doc.rust-lang.org/stable/std/primitive.i32.html#method.strict_rem_euclid)
```